test/include/common/harness.h

```diff
@@ -6,13 +6,10 @@
 
```

Comment:
nit: This looks like a lot of format changes for some reason, did we not run style formatter on test, or did we somehow change the style checks?

Reply:
"I also formatted the benchmark and test folders."